receiver/mongodbatlasreceiver/documentation.md

```diff
@@ -0,0 +1,130 @@
+ <li>ticket_type</li> </ul> |
+| mongodbatlas.system.cpu.normalized.usage | System CPU Normalized to pct Aggregates MongoDB Metrics MAX_SYSTEM_NORMALIZED_CPU_USER, MAX_SYSTEM_NORMALIZED_CPU_NICE, SYSTEM_NORMALIZED_CPU_IOWAIT, SYSTEM_NORMALIZED_CPU_GUEST, SYSTEM_NORMALIZED_CPU_IRQ, MAX_SYSTEM_NORMALIZED_CPU_IOWAIT, SYSTEM_NORMALIZED_CPU_KERNEL, SYSTEM_NORMALIZED_CPU_STEAL, MAX_SYSTEM_NORMALIZED_CPU_SOFTIRQ, MAX_SYSTEM_NORMALIZED_CPU_STEAL, SYSTEM_NORMALIZED_CPU_SOFTIRQ, MAX_SYSTEM_NORMALIZED_CPU_KERNEL, MAX_SYSTEM_NORMALIZED_CPU_GUEST, SYSTEM_NORMALIZED_CPU_NICE, SYSTEM_NORMALIZED_CPU_USER, MAX_SYSTEM_NORMALIZED_CPU_IRQ | 1 | Gauge | <ul>
+ <li>cpu_state</li> <li>aggregation</li> </ul> |
+| mongodbatlas.system.cpu.usage | System CPU Usage (%) Aggregates MongoDB Metrics SYSTEM_CPU_USER, MAX_SYSTEM_CPU_SOFTIRQ, MAX_SYSTEM_CPU_IRQ, MAX_SYSTEM_CPU_GUEST, MAX_SYSTEM_CPU_IOWAIT, SYSTEM_CPU_GUEST, SYSTEM_CPU_SOFTIRQ, MAX_SYSTEM_CPU_NICE, SYSTEM_CPU_IRQ, SYSTEM_CPU_KERNEL, SYSTEM_CPU_IOWAIT, SYSTEM_CPU_NICE, SYSTEM_CPU_STEAL, MAX_SYSTEM_CPU_KERNEL, MAX_SYSTEM_CPU_USER, MAX_SYSTEM_CPU_STEAL | 1 | Gauge | <ul>
+ <li>cpu_state</li> <li>aggregation</li> </ul> |
+| mongodbatlas.system.fts.cpu.normalized.usage | Full text search disk usage (%) Aggregates MongoDB Metrics FTS_PROCESS_NORMALIZED_CPU_USER, FTS_PROCESS_NORMALIZED_CPU_KERNEL | 1 | Gauge | <ul>
+ <li>cpu_state</li> </ul> |
+| mongodbatlas.system.fts.cpu.usage | Full-text search (%) | 1 | Gauge | <ul> <li>cpu_state</li> </ul> |
+| mongodbatlas.system.fts.disk.used | Full text search disk usage MongoDB Metric FTS_DISK_USAGE | By | Gauge | <ul>
+ </ul> |
+| mongodbatlas.system.fts.memory.usage | Full-text search Aggregates MongoDB Metrics FTS_MEMORY_MAPPED, FTS_PROCESS_SHARED_MEMORY, FTS_PROCESS_RESIDENT_MEMORY, FTS_PROCESS_VIRTUAL_MEMORY | MiBy | Sum | <ul>
+ <li>memory_state</li> </ul> |
+| mongodbatlas.system.memory.usage | System Memory Usage Aggregates MongoDB Metrics SYSTEM_MEMORY_AVAILABLE, MAX_SYSTEM_MEMORY_CACHED, MAX_SYSTEM_MEMORY_AVAILABLE, MAX_SYSTEM_MEMORY_USED, SYSTEM_MEMORY_BUFFERS, MAX_SYSTEM_MEMORY_BUFFERS, MAX_SYSTEM_MEMORY_FREE, SYSTEM_MEMORY_USED, SYSTEM_MEMORY_CACHED, SYSTEM_MEMORY_SHARED, MAX_SYSTEM_MEMORY_SHARED, SYSTEM_MEMORY_FREE | KiBy | Gauge | <ul>
+ <li>memory_state</li> <li>aggregation</li> </ul> |
+| mongodbatlas.system.network.io | System Network IO Aggregates MongoDB Metrics SYSTEM_NETWORK_IN, SYSTEM_NETWORK_OUT, MAX_SYSTEM_NETWORK_OUT, MAX_SYSTEM_NETWORK_IN | By/s | Gauge | <ul>
+ <li>direction</li> <li>aggregation</li> </ul> |
+| mongodbatlas.system.paging.io | Swap IO Aggregates MongoDB Metrics MAX_SWAP_IO_IN, SWAP_IO_IN, SWAP_IO_OUT, MAX_SWAP_IO_OUT | {pages}/s | Gauge | <ul>
+ <li>direction</li> <li>aggregation</li> </ul> |
+| mongodbatlas.system.paging.usage | Swap usage Aggregates MongoDB Metrics MAX_SWAP_USAGE_FREE, MAX_SWAP_USAGE_USED, SWAP_USAGE_FREE, SWAP_USAGE_USED | KiBy | Gauge | <ul>
+ <li>direction</li> <li>aggregation</li> </ul> |
+
+## Attributes
```

Comment:
Maybe "Datapoint Attributes" to help distinguish from resource?

Reply:
This document is autogenerated by `mdatagen`.